packages/docs/components/tag.md

```diff
@@ -117,89 +120,43 @@ As stated above, Tags are used to describe an entity or object, not physically r
       <thead>
         <tr>
           <th scope="column" class="is-ods-table-checkbox">
-            Item
+            Agency
+          </th>
+          <th scope="column" class="is-ods-table-num">
+            Formed
           </th>
-          <th scope="column" class="is-ods-table-num">Price</th>
         </tr>
       </thead>
       <tbody>
         <tr>
           <td>
             <ul class="ods-tag--list">
-              <li class="ods-tag">Glazed</li>
+              <li class="ods-tag">NASA</li>
             </ul>
           </td>
-          <td class="is-ods-table-num">$0.75</td>
-        </tr> 
-        <tr>
-          <td>
-            <ul class="ods-tag--list">
-              <li class="ods-tag">Chocolate glazed</li>
-            </ul>
-          </td>
-          <td class="is-ods-table-num">$0.90</td>
-        </tr> 
-        <tr>
-          <td>
-            <ul class="ods-tag--list">
-              <li class="ods-tag">Homer sprinkle</li>
-            </ul>
+          <td class="is-ods-table-num">
+            1958
           </td>
-          <td class="is-ods-table-num">$3.99</td>
-        </tr>
-      </tbody>
-    </table>
-  </figure>
-</Visual>
-
-### One or more
-
-<Description class="is-fpo">
-
-<span class="is-fpo is-fpo-negative">The following is figma guidance and needs to be re-written</span>
-
-If you need more than one Tag in your design, you can option drag the one you have for a duplicate (like anything else).
-
-Take a look at the anatomy section on the overview page. Keep in mind that they have a Small Spacing unit (6px) in between them.
-
-</Description>
-
-<Visual variant="positive">
```

Comment:
One possible issue with the current layout is that we can't choose to allow Visuals to cascade alongside content.

e.g. if we wanted the visuals here to live under `## Usage` but not row break on `### Describe an object, don’t be the object`

Reply:
Let's chat about this in the morning. I want Joe to have the opportunity to think through the problem. I do think two column visual makes sense, especially for usage - but will defer to him @joejezowski-okta

Reply:
@arnoldsandoval-okta you comfy merging this in before we solve for any layout changes? i'd like to keep these to content-only for the sake of velocity

Reply:
No objection from me.